app/src/main/res/values/strings.xml

```diff
@@ -217,4 +213,40 @@
      When the described button is pressed, the overlay will open and the browser will be hidden underneath it -->
     <string name="hint_press_menu_to_open_overlay_a11y">Press the menu button for %1$s Home, search, settings, and more</string>
 
+    <!-- FxA Strings-->
+    <!-- Content description for a button that will prompt a user to sign in, in order to receive tabs pushed from FF
+    on another device -->
+    <string name="fxa_login_a11y">Receive tabs</string>
+    <!-- Heading for settings menu item button and CTA link on new update -->
+    <string name="fxa_settings_item_new">New! Firefox tabs on Fire TV</string>
+    <!-- Heading for settings menu item button and CTA link when signed in -->
+    <string name="fxa_settings_item_signed_in">Sent tab settings</string>
+    <!-- Heading for settings menu item button and CTA link when need to sign in again -->
+    <string name="fxa_settings_item_sign_in_again">Sent tabs - sign in again</string>
+    <!-- FxA pre-boarding header -->
+    <string name="fxa_preboarding_header">See your tabs on the big screen</string>
+    <!-- FxA pre-boarding instruction text -->
+    <string name="fxa_preboarding_instruction">Send a tab from anywhere you use Firefox and see it here. Sign in to Firefox for Fire TV to get started.</string>
+    <!-- FxA pre-boarding sign in button -->
+    <string name="fxa_preboarding_primary_button">Sign in</string>
+    <!-- FxA pre-boarding not now button -->
+    <string name="fxa_preboarding_secondary_button">Not now</string>
+    <!-- FxA onboarding header -->
+    <string name="fxa_onboarding_header">Your tabs. Now ready for prime time.</string>
+    <!-- FxA onboarding instruction text -->
+    <string name="fxa_onboarding_instruction">You’re ready to receive tabs. To get started, select Send Tab to Device on Firefox for desktop and iOS or Share on Firefox Preview. Then choose Fire TV and your tab will appear in just a moment.</string>
+    <!-- FxA onboarding primary button -->
+    <string name="fxa_onboarding_primary_button">Ok, got it</string>
+    <!-- Toast shown on webpage when tab is sent from another device
+     %1$s will be replaced with the device name where the tab was sent from. -->
+    <string name="fxa_tab_sent_toast">Tab sent from %1$s</string>
+    <!-- FxA settings page header -->
+    <string name="fxa_settings_header">Sent tabs</string>
```

Comment:
Maybe we should call this, "sent_tabs_settings_header" because that's all that's in here right now. We can change it to, `fxa...` when it actually hosts more fxa data.

Reply:
Wouldn't it be easier to just leave the string names with `fxa`? Because I think if we change the name don't we have to get them re-localized?

Reply:
After your comment, I notice now that there are some strings that will likely need to be relocalized (e.g. `fxa_settings_header` = "Sent tabs") and some that will likely not be (e.g. `fxa_settings_body` = "Signed in as ...").

I say we stick with `fxa_*` so that, if this does because an FxA settings screen and we don't need to relocalize some strings, we don't have a mix of string identifiers with different prefix conventions (`sent_tab_*` vs `fxa_*`).

Thanks for raising this.